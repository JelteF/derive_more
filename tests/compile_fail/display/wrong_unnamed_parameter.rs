#[derive(derive_more::Display)]
#[display(fmt = "Stuff({_1})")]
pub struct Foo(String);

fn main() {}
