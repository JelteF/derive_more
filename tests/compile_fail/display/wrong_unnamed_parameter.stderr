error[E0425]: cannot find value `_1` in this scope
 --> tests/compile_fail/display/wrong_unnamed_parameter.rs:2:17
  |
2 | #[display(fmt = "Stuff({_1})")]
  |                 ^^^^^^^^^^^^^ help: a local variable with a similar name exists: `_0`
